test/test.js

```diff
@@ -294,8 +285,7 @@ describe('sourcemap-concat', function() {
     });
     builder = new broccoli.Builder(node);
     return builder.build().then(function(result) {
-      expectFile('nothing.js').in(result);
-      expectFile('nothing.map').in(result);
+      expectValidSourcemap('nothing.js').in(result);
```

Comment:
I just ran our empty source maps against sourcemaps.io and they are not complaining about the empty `sources` list.

see https://sourcemaps.io/validate?url=https%3A%2F%2Fgist.githubusercontent.com%2FTurbo87%2Fe39b740c872227698227d3fc7005ad36%2Fraw%2Fdc5f1ca29a8bb98030f63773cc2c4561026711ea%2Fnothing.js